autonomx/Generator.cpp

```diff
@@ -625,17 +647,19 @@ void Generator::applyInputRegion() {
                     inputValue = region->getIntensity();
                     if(inputValue >= 0.1) {
                         valueToWrite = 1;
-                    } else {
-                        valueToWrite = 0;
+                        writeLatticeValue(x, y, valueToWrite);
                     }
-                    qDebug() << "wolfram or GOL value: " << valueToWrite;
-                    writeLatticeValue(x, y, valueToWrite);
+
                 } else {
                     writeLatticeValue(x, y, region->getIntensity());
-                }
+                }           
+            }
             }
         }
     }
+
+double Generator::sigmoid(double intensity){
+    return intensity/(1+exp(-intensity));
```

Comment:
Can you put a space between each operator, and after the `for` and `if` keywords?

Reply:
Let's do that later. For now, I will simply approve it.